PF/Assignment2/PascalTringle.java

```diff
@@ -0,0 +1,68 @@
+public class PascalTringle {
+
+	public static String spaces(int row, int n) {
+
+		if (row < 2 * n && row > 0) {
+			String spaceOutput = "";
+			int space;
+			if (row > n) {
+				space = row - n;
+			} else {
+				space = n - row;
+			}
+
+			for (int count = 0; count < space; count++) {
+
+				spaceOutput = spaceOutput.concat(" ");
+			}
+
+			return spaceOutput;
+		} else {
+			return "wrong Input";
```

Comment:
Or you can simple return Null in this case. I think that will be fine rather than throwing exception.